pipeline-modules/infra-service/aws/terraform.yaml

```diff
@@ -147,6 +148,8 @@ template: |
         # save the result as input variable file for terraform commands
         - printenv | grep "{{ tf_var_name_prefix }}*" | sed 's/^{{ tf_var_name_prefix }}//' | awk -F '=' -v OFS='=' '{ gsub("{{ tf_var_name_dash_placeholder }}", "-", $1); st = index($0,"="); val=substr($0,st+1); print $1, val ~ /^{.*?}$/ || val ~ /^\[.*?\]$/ ? val:"\""val"\"" }' > {{ tf_var_file_path }}
         - cd "{{ local_code_path }}"
+        # run a pre script for user if user 
+        - if [ -f {{user_pre_script}} ] ; then chmod +x {{user_pre_script}} && ./{{user_pre_script}} ; fi
```

Comment:
I think we should update all cloud's terraform modules with the same change. Otherwise they will start diverging and it will become hard to keep track of which cloud has what functionality.

Reply:
Right. Will do that 👍👍

Reply:
While I was trying to do this for GCP, I realised, each step in GCP runs on its own fresh docker environment. So then when the pre script installs stuff, it'll not be available to next steps. This can be solved, but I was hoping if we merge the AWS update as a hotfix for Zendesk and work on other clouds seperately?

Reply:
Added VAN-2413 & VAN-2414 for updating GCP & Azure modules